plugins/woocommerce/changelog/fix-37599

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: fix
+
+Add a product name in the aria-label attribute on the cart page.
```

Comment:
Just a thought, what do you think about this for a changelog note instead? I think it's a little more descriptive

```suggestion
Add the product name to the "Remove from cart" button's aria-label in the cart and mini cart.
```